tests/snapshots/openapi.test.ts.md

```diff
@@ -0,0 +1,105 @@
+# Snapshot report for `tests/openapi.test.ts`
+
+The actual snapshot is saved in `openapi.test.ts.snap`.
+
+Generated by [AVA](https://avajs.dev).
+
+## Should generate the right definition with response object
+
+> Snapshot 1
+
+    `{␊
+    	"openapi": "3.0.0",␊
+    	"info": {␊
+    		"title": "my-service",␊
+    		"version": "1.0.0"␊
+    	},␊
+    	"paths": {␊
+    		"/my-3nd-controller": {␊
+    			"get": {␊
+    				"parameters": [],␊
+    				"responses": {␊
+    					"200": {␊
+    						"description": "Ok",␊
+    						"content": {␊
+    							"application/json": {␊
+    								"schema": {␊
+    									"$ref": "#/components/schemas/__type_Array_Entity"␊
```

Comment:
It should probably be `SuccessResponse_Array_Entity`

Reply:
i don't think i have access to the previous type since `resolve` is recursive ? i'm not familiar with the code but i'll look into it